src/hpc/threadapi.c

```diff
@@ -374,10 +374,10 @@ static pthread_mutex_t KeepAliveLock;
 
 Obj KeepAlive(Obj obj)
 {
-    Obj newKeepAlive = NewBag(T_PLIST, 4 * sizeof(Obj));
+    Obj newKeepAlive = NEW_PLIST(T_PLIST, 4);
```

Comment:
I like this. I think we should actually add even more `NEW_FOO` functions, and try to call `NewBag` directly as little as possible.